src/cache/file_cache_test.cc

```diff
@@ -155,7 +174,8 @@ TEST(FileCacheTest, RestoreSingleEntry) {
 
   ASSERT_TRUE(base::File::Write(object_path, expected_object_code));
   ASSERT_TRUE(base::File::Write(deps_path, expected_deps));
-  EXPECT_FALSE(cache.Find(code, cl, version, &entry1));
+  EXPECT_FALSE(
+      cache.Find(code, HashMap<int, Immutable>{}, cl, version, &entry1));
```

Comment:
Why to specify the full typename? - At least Clang 3.9.0 should compile fine an unqualified brace-initializer: `cache.Find(code, {}, cl, version, &entry1)` - IMHO, it looks shorter and cleaner.